coldfront/core/project/templates/project/project_request/savio/project_request_surveys_modal.html

```diff
@@ -0,0 +1,73 @@
+{% load crispy_forms_tags %}
+
+<div
+    id="survey-modal"
+    class="modal fade bd-example-modal-lg"
+    tabindex="-1"
+    role="dialog"
+    aria-labelledby="survey-modal"
+    aria-hidden="true">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
+      <div class="modal-header">
+        <p class="modal-title">Survey Responses</p>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+
+      <!-- survey form tabs -->
+      <div class="modal-nav">
+        <ul class='nav nav-tabs' role='tablist'>
+          {% for allocation_request, survey_form in survey_answers %}
+            {% if forloop.counter == 1 %}
+              <li id='survey-nav-item' class="nav-item"><a data-toggle="tab" class='nav-link active' href="#survey{{forloop.counter}}">Survey #{{forloop.counter}}</a></li>
+            {% else %}
+              <li id='survey-nav-item' class="nav-item"><a data-toggle="tab" class='nav-link' href="#survey{{forloop.counter}}">Survey #{{forloop.counter}}</a></li>
+            {% endif %}
+          {% endfor %}
+        </ul>
+      </div>
+
+      <!-- survey form contents -->
+      <div class="modal-body">
+        <div class="tab-content">
+          {% for allocation_request, survey_form in survey_answers %}
+            {% if forloop.counter == 1 %}
+              <div id="survey{{forloop.counter}}" class="tab-pane active" style="background: none">
+                <div class="card" style='margin-bottom: .5em'>
+                  <div class="card-body">
+                    <p class="card-text"><b>Requester:</b> {{ allocation_request.requester }} </p>
```

Comment:
For "Requester" and "Principal Investigator", display: "First Last (username)":

`{{ user.first_name }} {{ user.last_name }} ({{ user.username }})`